### PR TITLE
feat: add copy button to code block

### PR DESCRIPTION
**Pull request by rootial**
Fixes https://github.com/snapshot-labs/snapshot/issues/854

test in http://localhost:3000/#/strategy/contract-call
image

@ChaituVR @bonustrack

### CODE REVIEW

**Comment by mktcode** [comment]
@rootial Thanks a lot!

Btw:
> test in http://localhost:3000/#/strategy/contract-call

There are automatic deployments for every pull request. You can click here:

image

or here:

File: src/components/Ui/Markdown.vue [hunk 1]
```diff
@@ -19,6 +22,26 @@ const markdown = computed(() => {
   // body = sanitizeHtml(body);
   return body;
 });
+
+onMounted(() => {
+  document
+    .querySelector('.markdown-body')
+    .querySelectorAll('pre>code')
+    .forEach(function (code) {
```
Comment (mktcode) [inline src/components/Ui/Markdown.vue:30]: Good opportunity to add the missing border radius to the pre element.

```suggestion
    .forEach(function (code) {
      code.parentElement.classList.add('rounded-lg');
```

**Review by mktcode — APPROVED** [review]
Works well. :)

**Comment by rootial** [comment]
@mktcode thanks for your suggestion! update commited.

**Review by mktcode — APPROVED** [review]

**Comment by rootial** [comment]
@samuveth can you take a look.? thx

**Pull request merged**